README.md

```diff
@@ -18,7 +18,7 @@ It is **not** in the scope of the standard to:
 
 For more information:
 
-- Read the [full specification](SEARCHSPEC.md).
+- Read the [full specification](SPEC.md).
```

Comment:
In retrospect, this would have been a much better name for this file from day 1. The "search" part was already available from context.